modules/beta-private-cluster/scripts/wait-for-cluster.sh

```diff
@@ -25,12 +25,12 @@ CLUSTER_NAME=$2
 
 echo "Waiting for cluster $CLUSTER_NAME in project $PROJECT to reconcile..."
 
-current_status=$(gcloud container clusters list --project="$PROJECT" --filter=name:"$CLUSTER_NAME" --format="value(status)")
+current_status=$(gcloud container clusters list --project="$PROJECT" --filter=name:"$CLUSTER_NAME" --format="value(status)") --impersonate_service_account="$impersonate_service_account"
```

Comment:
I believe you will need to add an argument for the $impersonate_service_account
similar to here: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a48e6eaa7be1fadfa0191a980eccb9c2dab6cd0a/modules/beta-private-cluster/scripts/wait-for-cluster.sh#L24

Reply:
Thanks Bharath!! I missed out this. Will fix it